client/js/screens/Home/HomeContainer.js

```diff
@@ -1,16 +1,50 @@
 import React, { Component } from 'react';
-import { Platform, TouchableOpacity, View } from 'react-native';
+import { Platform, TouchableOpacity, View, Text } from 'react-native';
 import Ionicons from 'react-native-vector-icons/Ionicons';
-
+import { USER_QUERY, ALL_BEERS_QUERY } from '../../apollo/queries';
+import { Query } from 'react-apollo';
+import { getLoggedInUser } from '../../config/models';
 import Home from './Home';
 import styles from './styles';
 
 class HomeContainer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { viewerId: null };
+  }
+
+  componentDidMount = async () => {
+    const viewerId = await getLoggedInUser();
+    this.setState({ viewerId });
+  };
+
   static navigationOptions = {
     title: 'Home'
   };
+
   render() {
-    return <Home />;
+    return (
+      <Query query={USER_QUERY} variables={{ id: this.state.viewerId }}>
+        {({ loading, error, data }) => (
+          <Query
+            query={ALL_BEERS_QUERY}
+            variables={{ id: this.state.viewerId }}
```

Comment:
you should not need 2 queries here. If needed change your schema to get both results with a single query.